docs/data/data_lifecycle.md

```diff
@@ -0,0 +1,65 @@
+
+Sometimes, climate data is published online but not publically accessible, i.e. it requires some sort of credentials / authorization to access. Even if this is the case, programmatic access is often supported by tooling from the data providers. For example, the Copernicus Data ecosystem has their own API and dataset licenses. Any user-specific access credentials or tokens stored on User Directories are not accessible to other members of the hub; they remain private. However, they can be viewed by LEAP Data and Compute admins; if this is an issue, please consult us for alternative ingestion options.
+
+### Migrating HPC Data
+
+Many LEAP Scientists also have access to an HPC or external filesystem on which their data is being generated. The biggest reason to ingest such data is for collaboration.
+If the data is located behind a firewall on an HPC center, the normal 'pull' based paradigm of our feedstocks will not work. In this case we have an option to 'push' the data to a special "inbox" bucket (`'leap-pangeo-inbox'`) on the OSN Pod. Once the data is in this accessible intermediate staging area, the process and tools documented on the [feedstock template](https://github.com/leap-stc/LEAP_template_feedstock) ought to work.
+
+The biggest barrier here is external authentication; see the [Authentication section](../technical-reference/authentication.md) for how to do this. Once authenticated, users can initiate a data transfer from their 'local' machine (laptop, server, or HPC Cluster) with the tools documented below.
+
+## Data Tooling
+
+There are many tools available to interact with cloud object storage. We currently have basic operations documented for three tools:
+
+- GCloud SDK. One can interact with Google Cloud storage directly using the Google Cloud SDK and Command Line Interface. Please consult the [gcloud docs](https://cloud.google.com/sdk/docs/install).
+
+- [fsspec](https://filesystem-spec.readthedocs.io/en/latest/) (and its submodules [gcsfs](https://gcsfs.readthedocs.io/en/latest/) and [s3fs](https://s3fs.readthedocs.io/en/latest/)) provide filesystem-like access to local, remote, and embedded file systems from within a python session. Fsspec is also used by xarray under the hood and so integrates easily with normal coding workflows (and tools like Dask).
+
+  - `fsspec` can be used to transfer small amounts of data to google cloud storage or OSN. If you have more than a few hundred MB's, it is worth using Rclone
+  - `fsspec` should be installed by default on the **Base Pangeo Notebook** environment on the Jupyter-Hub. If you are working locally, you can install it with `pip or conda/mamba`. ex: `pip install fsspec`.
+
+- [rclone](https://rclone.org/) which provides a Command Line Interface to many different storage backends, see [here](../technical-reference/rclone.md) for more details.
+
+### Tool Selection
```

Comment:
This seems kinda redundant - we use `rclone` in almost all cases. Maybe it's easier to call out the single exception?